grpc-server/src/main/scala/com/avast/sst/grpc/server/GrpcServerConfig.scala

```diff
@@ -0,0 +1,11 @@
+package com.avast.sst.grpc.server
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration.Duration
+
+final case class GrpcServerConfig(port: Int,
+                                  handshakeTimeout: Duration,
+                                  maxInboundMessageSize: Int = 4 * 1024 * 1024,
+                                  maxInboundMetadataSize: Int = 8192,
```

Comment:
Are these limits the same as defaults in the underlying library?

Reply:
Maybe it could be better to make these parameters `Option`al so we wouldn't have to copy these defaults?

Reply:
`copy these defaults` where? :thinking:

Reply:
My question is if these defaults have been copied from the underlying library.

Reply:
Yes, they have been copied from the underlying library. I feel this solution is not perfect but I think it's better than to have everything as `Option` so that's why I've decided to do it that away.